language/Syntax.md

```diff
@@ -451,6 +451,12 @@ Operator sections also work for functions used this way:
 fooBy2 = (_ `foo` 2)
 ```
 
+Infix operators created using backticks are left associative with the highest precedence by default.
```

Comment:
PureScript does not allow you to change the fixity of infix operators, so I don't think we need to say "by default".